docs/src/static-pages/accessibility.md

```diff
@@ -383,21 +434,39 @@ Some people may struggle to see content unless they are able to magnify it.
 - Users of localised content
 
 ### **How**
-- Web: Allow users to zoom to 400% without losing content or functionality
-- App: Respect system text size preferences without losing content or functionality
+- **Web**: Allow users to zoom to 400% without losing content or functionality
+- **App**: Respect system text size preferences without losing content or functionality
 
-### **Do / Don't**
-- Do use good responsive design
-- Don't use elements that won't zoom well, like sticky footers
 
 ### **Tips**
-- "Sticky" UI can make parts of the screen inaccessible, especially on smaller screens
-- Web: Use responsive layout
-- App: Use `autolayout`
+
+- **Web**: 
+  - Use responsive layout
+- **App**:
+  - Use `autolayout`
+  - "Sticky" UI can make parts of the screen inaccessible, especially on smaller screens
 
 ### **Tests**
 - Magnify content up to 400% and check that all elements are still visible and operational across all devices and all screen sizes
 
+## Small screen size
+
+### **Why**
+
+Some people use mobiles with screens as small as `320px` wide
```

Comment:
```suggestion
Some people use devices with screens as small as `320px` wide
```

Reply:
I know this is correct, we should probably get it updated on the source docs so we don't deviate and get confused with the correct source of truth